addons/test_mail/tests/test_message_post.py

```diff
@@ -798,6 +800,39 @@ def test_message_post(self):
                 partner_ids=self.partner_portal.ids,
             )
 
+    @mute_logger('odoo.addons.mail.models.mail_mail', 'odoo.models.unlink')
+    @users('employee')
+    def test_message_post_using_controller(self):
+        """ Test mail_message_post of ThreadController (especially on archived record). """
+        # Use an archivable test record
+        test_record = self.env['mail.test.activity'].create({})
+        body_text = 'Log a note on record'
```

Comment:
Would add an html input, we could check it is effectively keps and not escaped while we are here